neoload/neoload_cli_lib/docker_lib.py

```diff
@@ -29,9 +29,20 @@
     DOCKER_LAUNCHED: []
 }
 
-client = docker.from_env()
+client=None
 
 
+def get_docker_client():
+    global client
+    if not client:
+        try:
+            client = docker.from_env()
+        except Exception as ex:
```

Comment:
The variable "ex" is never used... Sonar tells it's a code smell. 
The rest is good ! I can validate as soon as it's fixed

Reply:
After applying this specific suggestion, code coverage on *new lines is still 79.2% (almost all is exception handling lines), so I'm going to merge and push a new release to get out asap.